getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,99 @@
+# Data Node Authentication [](distdb-auth)
+
+In a multi-node environment, it is necessary to ensure that the access
+node have the necessary roles and permissions on all the data nodes
+where distributed hypertables are placed.
+
+This is critical since when TimescaleDB is executing a query or
+command it will use the same role on the data nodes as on the access
+node. This means that the same roles need to exist on the data nodes
+and the access node need to have authentication set up for those
+roles.
+
+In a typical setup, you will create the distributed hypertable in a
+database and use the same database on all the data nodes. This means
+that you need to ensure that a user that is connected to the access
+node can access the same databases on the data nodes.
+
+>:NOTE: Setting up a secure system is a complex task and this section
+> should not be read as recommending any particular security measures
+> for securing your system. The chapter contains technical
+> instructions for how to set up the different authentication methods.
+
+## Setting up Password Authentication [](distdb-auth-pass)
+
+The easiest authentication to set up is password
+authentication. Configuring password authentication between the access
+node and data nodes is a two step process:
+
+1. Each data node needs to be configured to accept and authenticate
+   connections using passwords.
+
+2. The access node needs data node passwords for each role using the
+   access node. We will deal with each step separately below.
+
+In this section, the focus is on setting up SCRAM SHA-256 password
+authentication. For other password authentication methods, [see the
+PostgreSQL manual][auth-password].
+
+>:NOTE: The `password` method should not be used since it sends the
```

Comment:
I am slightly concerned about this section, since SCRAM authentication is a really a new way to do it and it does not really supported by a connection poolers yet. Most of folks live with `md5` and use it likely with TLS. I hope it will not confuse people

Reply:
Yeah, good point, but I think there are dangers in using MD5 in our examples if people copy it and think it is safe. If they run into this problem and think a little, they should be fine.